app/assets/stylesheets/base/_base.scss

```diff
@@ -13,3 +13,15 @@
 @import "lists";
 @import "tables";
 @import "typography";
+
+a {
+  .b {
+    .c {
+      .d {
+        .e {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5